kvmagent/kvmagent/plugins/ovn.py

```
@@ -413,6 +495,116 @@ def ovn_set_controller_connection(self, req):
 
         vsctl = ovn.VsCtl()
         vsctl.setOvsExternalIds("ovn-remote", conn_str)
+
+    @bash.in_bash
+    def ovn_check_local_port(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
```

Ruff (0.14.7)

Local variable `cmd` is assigned to but never used

Remove assignment to unused variable `cmd`

(F841)